### ISSUE TITLE
Do not issue warning if this number is 100%

### ISSUE THREAD

**Issue by JoseEmilio-ARM** [KhronosGroup/Vulkan-ValidationLayers#9732]
https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/a5db986d251b207fa241d7ea3b53c359618c3b54/layers/best_practices/bp_drawdispatch.cpp#L306

Otherwise it results in unhelpful messages like:
```
[PERFORMANCE WARN:] Validation Performance Warning: [ BestPractices-Arm-vkCmdDrawIndexed-sparse-index-buffer ] | MessageID = 0x8adbf7e3 | vkCmdDrawIndexed():  [Arm] The indices which were specified for the draw call only utilise approximately 100.00% of index buffer value range. Arm Mali architectures before G71 do not have IDVS (Index-Driven Vertex Shading), meaning all vertices corresponding to indices between the minimum and maximum would be loaded, and possibly shaded, whether or not they are used.
```
